src/pages/index.tsx

```diff
@@ -103,6 +120,19 @@ const HomePage: NextPageWithLayout = () => {
       });
     }
   }, [signedIn]);
+  useEffect(() => {
+    if (iOSDevice) {
+      setHomeScreenApp(window.matchMedia('(display-mode: standalone)').matches);
+    }
+  }, [iOSDevice]);
+
+  useEffect(() => {
+    if (iOSDevice) {
+      window.matchMedia('(display-mode: standalone)').addEventListener('change', (e) => setHomeScreenApp(e.matches));
+    }
+  }, [iOSDevice]);
```

Comment:
These lines checks does user uses a home screen app. Here is a [doc](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode)

Reply:
Is it necessary that lines 125 and 131 exist in distinct useEffects?

Reply:
Yes. This is the same functionality as for [Navigation](https://github.com/near/near-discovery/blob/develop/src/components/navigation/Navigation.tsx#L9-L15) component where we checking mobile/desktop device